src/test/java/org/jitsi/meet/test/StartMutedTest.java

```diff
@@ -261,4 +261,102 @@ public void startWithAudioMutedCanUnmute()
             "participant1",
             false);
     }
+
+    /**
+     * Join the conference while both audio and video are muted, unmute both the participants.
+     */
+    @Test(dependsOnMethods = { "startWithAudioMutedCanUnmute" })
+    public void startWithAudioVideoMutedCanUnmute()
+    {
+        hangUpAllParticipants();
+
+        JitsiMeetUrl meetUrl = getJitsiMeetUrl().appendConfig(
+                "config.startWithAudioMuted=true&" +
+                "config.startWithVideoMuted=true&" +
+                "config.p2p.enabled=true");
+
+        // Do not use ensureTwoParticipants() because it checks for send/rec
+        // bitrate which should be 0 if both participants start audio muted.
+        ensureOneParticipant(meetUrl);
+
+        WebParticipant participant1 = getParticipant1();
+        WebParticipant participant2 = joinSecondParticipant(meetUrl);
+
+        participant2.waitToJoinMUC();
+        participant2.waitForIceConnected();
+
+        MeetUIUtils.waitForAudioMuted(
+            participant1.getDriver(),
+            participant2.getDriver(),
+            "participant2",
+            true);
+
+        MeetUIUtils.waitForAudioMuted(
+            participant2.getDriver(),
+            participant1.getDriver(),
+            "participant1",
+            true);
+
+        participant1.getParticipantsPane().assertIsParticipantVideoMuted(participant2, true);
+
+        participant2.getParticipantsPane().assertIsParticipantVideoMuted(participant1, true);
+
+        // Unmute p1's both audio and video and check on p2.
+        participant1.getToolbar().clickAudioUnmuteButton();
+        MeetUIUtils.unmuteVideoAndCheck(participant1, participant2);
+
+        participant2.getLargeVideo().isVideoPlaying();
+
+        MeetUIUtils.waitForAudioMuted(
+            participant2.getDriver(),
+            participant1.getDriver(),
+            "participant1",
+            false);
+
+        // Unmute p2's audio and video and check on p1.
+        MeetUIUtils.unmuteVideoAndCheck(participant2, participant1);
+        participant2.getToolbar().clickAudioUnmuteButton();
+
+        participant1.getLargeVideo().isVideoPlaying();
```

Comment:
Are you using this for a side effect or to assert something? The doc says it just returns a boolean

Reply:
Oops, I just copy pasted from the existing testcase, will fix it.